learning/katas/go/Core Transforms/GroupByKey/GroupByKey/pkg/task/task.go

```diff
@@ -0,0 +1,27 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import "github.com/apache/beam/sdks/go/pkg/beam"
+
+func ApplyTransform(s beam.Scope, input beam.PCollection) beam.PCollection {
+	kv := beam.ParDo(s, splitByFirstChar, input)
+	return beam.GroupByKey(s, kv)
+}
+
+func splitByFirstChar(element string) (uint8, string) {
```

Comment:
If we return using rune, the output might not be as intuitive.
I suggest that we return as KV<string, string>.
![image](https://user-images.githubusercontent.com/5459430/82194481-78f12580-9929-11ea-8c95-7f7db90e8e61.png)